manifests/params.pp

```diff
@@ -377,8 +377,8 @@
   $server_connect_timeout          = 120000
   $server_enable_ruby_profiler     = false
   $server_ca_auth_required         = true
-  $server_admin_api_whitelist      = [ '127.0.0.1', '::1', $::ipaddress ]
```

Comment:
Note: foreman-installer will need a migration to replace this value, as `::1` causes the server to fail to start (`Exception in thread "main" java.lang.Exception: invalid domain value: ::1`)